examples/graphql-helix-auth0/index.ts

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable no-console */
+import fastify from 'fastify';
+import { getGraphQLParameters, processRequest, renderGraphiQL, shouldRenderGraphiQL } from 'graphql-helix';
+import { envelop, useSchema } from '@envelop/core';
+import { useAuth0 } from '@envelop/auth0';
+import { makeExecutableSchema } from '@graphql-tools/schema';
+
+const schema = makeExecutableSchema({
+  typeDefs: /* GraphQL */ `
+    """
+    Describes the authentication object as proivided by Auth0.
+    """
+    type AuthenticationInfo {
+      """
+      String that uniquely identifies an authenticated user.
+      """
+      sub: String!
+    }
+
+    type Query {
+      """
+      The authentication information of the request.
+      """
+      authInfo: AuthenticationInfo
```

Comment:
Maybe this could be `type Query { viewer: User! } type User { id: ID! }` (and then map it to `auth0.sub`) ?

Reply:
I wanted to reduce complexity by avoiding doing complex mappings within the resolvers. At the end in the `Next steps` section, I briefly mention this.